### PR TITLE
Add console based user interaction

### PR DESCRIPTION
**Pull request by tkz96**
This PR is to:

- Add console based user interactivity to the app

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+    run
+  end
+end
+
+def main
+  app = App.new
+  app.run
+end
+
+main
```
Comment (Ismailco) [inline app.rb:178]: - [ ] Kindly consider creating a `main.rb` file and add your entry point `main` there so you can run the app by calling the main.rb (ex: `ruby main.rb`)

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+  def show_menu
+    puts "Welcome to School Library App!\n\n"
+    puts 'Please choose an option by entering a number:'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all rentals for a given person id'
+    puts "7 - Exit\n\n"
```
Comment (Ismailco) [inline app.rb:23]: - [ ] Your console app should behave in the same way as it is in the video example, if the user chooses an option kindly consider returning the output but don't exit the app until the user selects option 7. as for now the app exit after returning  anything from the other options 

Peek 2022-08-06 15-28

File: app.rb [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+  end
+
+  def show_menu
+    puts "Welcome to School Library App!\n\n"
+    puts 'Please choose an option by entering a number:'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
```
Comment (Ismailco) [inline app.rb:21]: - [ ] Your methods for `Creating person`, `Creating book`, and `Creating rentals` are not working as expected, kindly fix this so your app is working properly

The output from the terminal:
Screenshot_20220806_153010
Screenshot_20220806_153032
Screenshot_20220806_152929

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @tkz96, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by EricMbouwe — APPROVED** [review]
Hi @tkz96, 

Your project is complete! ⚡🔥  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🎉🔥
- Linters ✔
- PR ✔
- Requested changes ✔
## Optional suggestions ☂

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**